### PR TITLE
Handle discussion IDs using Facia client instead of Frontend

### PR DESCRIPTION
**Pull request by buck06191**
## What does this change?
Bump Scala Facia client version to v2.3.0. This version introduces a fix for the new short URL format. As a result we can remove the temporary fix introduced in #23579 .

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by buck06191** [comment]
Checked on CODE and all works

File: project/Dependencies.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ object Dependencies {
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.23"
   val contentApiClient = "com.gu" %% "content-api-client" % capiVersion
   val dfpAxis = "com.google.api-ads" % "dfp-axis" % "4.12.0"
-  val faciaFapiScalaClient = "com.gu" %% "fapi-client-play27" % faciaVersion
+  val faciaFapiScalaClient = "com.gu" %% "fapi-client-play26" % faciaVersion
```
Comment (jfsoul) [inline project/Dependencies.scala:35]: how come we're downgrading play to 2.6?

Comment (buck06191) [reply project/Dependencies.scala:35]: Ah that's a rebase error. I can fix.

Comment (buck06191) [reply project/Dependencies.scala:35]:
```suggestion
  val faciaFapiScalaClient = "com.gu" %% "fapi-client-play27" % faciaVersion
```

**Review by jfsoul — APPROVED** [review]

**Pull request merged**